modules/ensemble/lib/widget/input/dropdown.dart

```diff
@@ -246,6 +249,8 @@ class SelectOneController extends FormFieldController with HasTextPlaceholder {
   SelectOneInputFieldAction? inputFieldAction;
   List<SelectOneItem>? items;
 
+  TextEditingController textEditingController = TextEditingController();
```

Comment:
this should be handled internally in the widget, there is no need to expose it in the controller

Reply:
I needed to add it to the controller so that its available in the setters

Reply:
you don't need it to be available in the setters. You can update the value in the onchange event inside the state.

Reply:
what happens when user selects a value from the dropdown? You are setting the value when it is set in setters but what about when it is selected by user interaction? Will the textEditingController update, I don't see the code?